trpcultivate_phenotypes/src/Plugin/Validators/DuplicateTraits.php

```diff
@@ -135,10 +171,12 @@ public function validateRow($row_values) {
     }
 
     // Check if our trait combo exists at the database level
-    // First make sure to set our genus before getting our trait combo
-    $this->service_traits->setTraitGenus($context['genus']);
+    // First make sure to get our genus and then set it in our Traits Service
+    // before we can grab our trait combo
+    $genus = $this->getConfiguredGenus();
+    $this->service_PhenoTraits->setTraitGenus($genus);
```

Comment:
Is this actually needed now that we use the trait? Or is this done by the trait when you configure the genus?

Reply:
It is still needed for tests to pass. 🤷  TraitImporterFormValidateTest() throws this exception if I comment it out: 
```
Exception : No genus has been set. See setting a genus in the Traits Service and make sure to
        use a configured genus. To configure a genus or see all configured genus, go to /admin/tripal/extension/tripal-cultivate/phenotypes/ontology.
```